RevenueCatUI/CustomerCenter/Views/CustomerCenterNavigationLink.swift

```diff
@@ -49,24 +57,77 @@ public struct CustomerCenterNavigationLink<Label: View>: View {
     /// }
     /// ```
     ///
-    /// - Parameter label: A closure that returns the view to display as the navigation link’s label.
+    /// - Parameter label: A closure that returns the view to display as the navigation link's label.
+    @available(*, deprecated, message: """
+    Use the view modifiers instead.
+    For example, use .onCustomerCenterRestoreStarted(), .onCustomerCenterRestoreCompleted(), etc.
+    """)
     public init(
-        customerCenterActionHandler: CustomerCenterActionHandler? = nil,
+        customerCenterActionHandler: CustomerCenterActionHandler?,
```

Comment:
won't this break the public API? 👀

Reply:
It won't because there's now the new init
```
public init(@ViewBuilder label: @escaping () -> Label)
```

So doing 
```
CustomerCenterNavigationLink {

}
```
now uses the new, nondeprecated `init`.

Reply:
Nice